README.md

```diff
@@ -1 +1,189 @@
-# appointment-app-back-end
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

Reply:
Implemented above suggested change - test folder has been removed